Notebooks/antibody_epitopes/causal_analysis.ipynb

```diff
@@ -160,19 +160,19 @@
        "((61, 11), (61, 11))"
       ]
      },
-     "execution_count": 13,
+     "execution_count": 10,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "df_dead_late = df_dead_late.sample(61)\n",
+    "df_dead_late = df_dead_late.sample(61) # This randomly selects 61 Tx'es from the late group (n=66), which will cause it to be slightly different each time the notebook is rerun\n",
```

Comment:
This is causing the "Late failure" KM plot to be different each time the notebook is rerun. The difference can be quite large because the groups are already so small (e.g., the p-value seems to fluctuate between .5 and .9)

What's the rationale for doing this? Should I just remove it and use the full 66 transplants?

Reply:
Since there seemed to be no immediate use for this I've removed it now in [38cc933](https://github.com/umcu-stride/desa_paper/pull/4/commits/38cc93353021a613af4542dbbed2c0df846ab49e)

Reply:
Thanks Leon for your contribution. Sorry, for the late response. 
No problem with removing it. The idea was to balance the treatment and control group so that the difference would be significance. However, due to small sample size, I can imagine some samples with large outcome could cause noise and inconsistency in each try.